pkg/option/config.go

```diff
@@ -3123,6 +3155,59 @@ func (c *DaemonConfig) calculateDynamicBPFMapSizes(totalMemory uint64, dynamicSi
 	}
 }
 
+// Validate VTEP support configuration
+func (c *DaemonConfig) validateVTEP() error {
+	vtepEndpoints := viper.GetStringSlice(VtepEndpoint)
+	vtepCIDRs := viper.GetStringSlice(VtepCIDR)
+	vtepMACs := viper.GetStringSlice(VtepMAC)
+
+	if (len(vtepEndpoints) < 1) ||
+		len(vtepEndpoints) != len(vtepCIDRs) ||
+		len(vtepEndpoints) != len(vtepMACs) ||
+		len(vtepCIDRs) != len(vtepMACs) {
+		err := errors.New("VTEP config number does not match")
+		log.WithError(err).WithFields(logrus.Fields{
+			"vtep": len(vtepEndpoints),
+		}).Error("Endpoints or CIDRs or MACs number does not match")
+		return err
+	}
+	for i := 0; i < len(vtepEndpoints); i++ {
+		endpoint := net.ParseIP(vtepEndpoints[i])
+		if endpoint == nil {
+			err := errors.New("Unable to parse VTEP endpoint")
+			log.WithError(err).WithFields(logrus.Fields{
+				"allocEndpoint": vtepEndpoints[i],
+			}).Error("Unable to parse VTEP endpoint")
+			return err
```

Comment:
For these ones you can just do:
```suggestion
			return fmt.Errorf("Invalid VTEP IP: %v", vtepEndpoints[i])
```

Reply:
done